### DISCUSSION TITLE
Share styling amongst typst documents

### DISCUSSION THREAD

**Discussion by frank-ruegamer** [typst/typst#1446] [Q&A] [answered]
Say I have two files with the following content

**document.typ**
```typ
#include "preamble.typ"
= Chapter 2
```
**preamble.typ**
```typ
#set page(paper: "a6")
= Chapter 1
```

What comes out is the following:
[document.pdf](https://github.com/typst/typst/files/11706331/document.pdf)

two problems with that:
1. only the first page will be in A6
2. Chapter 2 will be on a new page (this will be fixed by removing the `#set` line from `preamble.typ`

I also tried to use `import` to no avail 

=> how does one extract styling in to a separate file?

What I ultimately want to do is produce two almost identical documents with some changed content in between them.

**Comment by laurmaedje** [comment, marked as answer]
This might be of help: https://typst.app/docs/tutorial/making-a-template/
